packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -31,7 +31,7 @@ class FormAds extends Component {
 
     componentDidMount() {
         // TODO: [p2p-fix-api] call get offer detail api and populate state
-        WS().send({ 'residence_list': 1 }).then(() => { // this is just to mock the api delay response
+        WS({ 'residence_list': 1 }).then(() => { // this is just to mock the api delay response
```

Comment:
Is this comment still valid?

Reply:
yes, thats not valid, and we should not send residence_list too. thank you for the catch :pray: